### PR TITLE
Footer css

### LINKED ISSUE #79: Chibuzo- Develop Footer with TailwindCSS
Using Tailwind CSS to style the footer component

### PR DESCRIPTION
**Pull request by Domicaren1**
## What type of PR is this? (check all applicable)

- [ ] Feature
- [x] Refactor
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Other (please specify here: <type here>)
  
  
## Description of what this Pull Request does
Style footer component

## Related Tickets & Documents
- Closes #79 
- Related Issues: (enter ticket numbers here)
  
## QA Instructions, Screenshots, Recordings
(In this section, describe how to QA/review your PR. Feel free to use screenshots and/or short videos)
  
chibuzo footer css

## Added/updated tests?
- [ ] Yes
- [ ] Not Yet, and here is why: __explain here__
- [ ] I need help with tests
- [x] No, and here is why: __explain here__

### CODE REVIEW

**Comment by TaylorDJones11** [comment]
@Domicaren1 check this PR, this is no need to add or remove anything from the Tailwind config file. Everything is already there

**Review by jjleigh — APPROVED** [review]

**Comment by Domicaren1** [comment]
@TaylorDJones11 I didn't touch the tailwind config file. will check now.

**Review by ibimina — CHANGES_REQUESTED** [review]
@Domicaren1 I have attached a screenshot of your pr and a description.

- The color of the text doesn't match with the design please fix up
- The copyright section is also missing in the footer component

Footer component should look like this
footerdesign

**Comment by Domicaren1** [comment]
@TaylorDJones11 pls can you include this color code to the tailwind config :#3181DF;
Thanks.

**Comment by Domicaren1** [comment]
@ibimina the copyright section isn't included in the code.

**Comment by TaylorDJones11** [comment]
> @TaylorDJones11 pls can you include this color code to the tailwind config :#3181DF and #1e1e20 Thanks.

Sure. Where are these hex codes coming from? Irene's Figma file?

**Comment by Domicaren1** [comment]
@TaylorDJones11 yes from Irene's figma file. For the logo text color and box shadow.

**Comment by msarit** [comment]
@Domicaren1 this branch is severely behind `main` branch.
Please run the command `git pull origin main` to update your branch with `main`.

File: src/components/common/FooterBar.js [hunk 1]
```diff
@@ -8,8 +8,8 @@ function FooterBar() {
   ];
 
   return (
-    <footer className="bg-blue-1000 ">
-      <span className="text-blue-500 uppercase text-s mt-6">Logo</span>
+    <footer className="bg-primary-dark flex justify-between ">
+      <span className="text-gradient-dark-blue uppercase text-s mt-6">Logo</span>
```
Comment (msarit) [inline src/components/common/FooterBar.js:12]: @Domicaren1 I do not think this is the correct shade of blue. Please check the tailwind.config file for a more accurate shade. Don't worry about the links - the team decided not to have those pages, so those links will probably be deleted.

**Review by msarit — CHANGES_REQUESTED** [review]

File: src/components/common/FooterBar.js (hunk 1 above)
Comment (Domicaren1) [reply src/components/common/FooterBar.js:12]: @msarit It's corrected already ma

**Review by ibimina — APPROVED** [review]
LGTM!

**Review by ibimina — APPROVED** [review]

**Review by msarit — APPROVED** [review]

**Pull request merged**